### ISSUE TITLE
Builds aren't added to commit

### ISSUE THREAD

**Issue by hollowdoor** [observing/pre-commit#81]
What could be done to make sure build files created by pre-commit are added to the stage?

Is this something that can be in pre-commit, or is there something I can do on my end?

For instance I could just add a command to the package.json scripts that stages the built files on pre commit.

**Comment by rpribadi** [comment]
Facing the same situation. What's the recommendation for this?
Thanks!

**Comment by barvian** [comment]
I agree this would be a nice options. For now I'm just doing something like:

```
{
  "scripts": {
    ...
    "build:add": "git add [whatever files related to build]"
  },
  "precommit": [
    "clean",
    "bower",
    "lint",
    "build",
    "build:add"
  ]
}
```
